packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -197,10 +205,11 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
   @override
   Widget build(BuildContext context) {
     final bool closed = !_isExpanded && _controller.isDismissed;
+    final bool shouldRemoveChildren = closed && !widget.maintainsState;
     return AnimatedBuilder(
       animation: _controller.view,
       builder: _buildChildren,
-      child: closed ? null : Column(children: widget.children),
+      child: shouldRemoveChildren ? null : Column(children: widget.children),
```

Comment:
Just keeping them in the tree like this doesn't seem like a good idea. Animations will keep running and drain the user's battery even though nothing is visible. We should turn animations off and maybe even put the non-visible children offstage to ensure that you can't interact with them. For an example, look at how the "Visibility" widget implements the maintainState flag.

Reply:
Thanks for pointing to the Visibility widget! Should I wrap the children in a Visibility widget or instead wrap it in both Offstage and TickerMode?